app/lib/feed_manager.rb

```diff
@@ -42,7 +42,7 @@ def filter(timeline_type, status, receiver)
     when :home
       filter_from_home(status, receiver.id, build_crutches(receiver.id, [status]), :home)
     when :list
-      (filter_from_list?(status, receiver) ? :filter : nil) || filter_from_home(status, receiver.account_id, build_crutches(receiver.account_id, [status]), :list)
+      (filter_from_list?(status, receiver) ? :filter : nil) || filter_from_home(status, receiver.account_id, build_crutches(receiver.account_id, [status], list: list), :list)
```

Comment:
Where is the value of this newly added list set? (It doesn't seem to be set.)
Maybe the `receiver`.

Reply:
Oops, great catch! Going to fix this.